ngraph/test/backend/deformable_convolution_opset8.in.cpp

```diff
@@ -0,0 +1,2940 @@
+// Copyright (C) 2021 Intel Corporation
```

Comment:
I have 5 failed tests on IE_GPU on local run. Is it expected?
[  FAILED  ] 5 tests, listed below:
[  FAILED  ] IE_GPU.deformable_convolution_opset8_2D_integral_offsets_groups_and_deforgroups
[  FAILED  ] IE_GPU.deformable_convolution_opset8_2D_v8_zeroed_offsets_default_modulation_scalars
[  FAILED  ] IE_GPU.deformable_convolution_opset8_2D_zeroed_offsets_groups_and_deforgroups_modulation_scalars
[  FAILED  ] IE_GPU.deformable_convolution_opset8_2D_zeroed_offsets_groups_and_deforgroups_modulation_scalars_2
[  FAILED  ] IE_GPU.deformable_convolution_opset8_2D_zeroed_neg_offsets_groups_and_deforgroups_modulation_scalars

Reply:
*_modulation_scalars are expected, they will be added in skip list
deformable_convolution_opset8_2D_integral_offsets_groups_and_deforgroups shouldn't fail, I will look at this

Reply:
deformable_convolution_opset8_2D_integral_offsets_groups_and_deforgroups - it's a bug, created a ticket on GPU plugin